railstats.py

```
@@ -144,12 +149,18 @@ def get_data(data, counts):
     for operator in counts['operator']:
         counts['percent_delayed_by_operator'][operator] = 0.0
         if 'late' in counts['arrival_status_by_operator'][operator]:
-            counts['percent_delayed_by_operator'][operator] = counts['arrival_status_by_operator'][operator]['late'] / counts['operator'][operator]
+            try:
+                counts['percent_delayed_by_operator'][operator] = counts['arrival_status_by_operator'][operator]['late'] / (counts['arrival_status_by_operator'][operator]['late'] + counts['arrival_status_by_operator'][operator]['early'] + counts['arrival_status_by_operator'][operator]['RT'])
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)